datasets/numeric_fused_head/numeric_fused_head.py

```diff
@@ -0,0 +1,141 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NFH: Numeric Fused-Heads."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+
+import datasets
+
+
+_CITATION = """\
+@article{elazar_head,
+    author = {Elazar, Yanai and Goldberg, Yoav},
+    title = {Where’s My Head? Definition, Data Set, and Models for Numeric Fused-Head Identification and Resolution},
+    journal = {Transactions of the Association for Computational Linguistics},
+    volume = {7},
+    number = {},
+    pages = {519-535},
+    year = {2019},
+    doi = {10.1162/tacl\\_a\\_00280},
+    URL = {https://doi.org/10.1162/tacl_a_00280},
+}
+"""
+
+_DESCRIPTION = """\
+Fused Head constructions are noun phrases in which the head noun is \
+missing and is said to be "fused" with its dependent modifier. This \
+missing information is implicit and is important for sentence understanding.\
+The missing heads are easily filled in by humans,  but pose a challenge for \
+computational models.
+
+For example, in the sentence: "I bought 5 apples but got only 4.", 4 is a \
+Fused-Head, and the missing head is apples, which appear earlier in the sentence.
+
+This is a crowd-sourced dataset of 10k numerical fused head examples (1M tokens).
+"""
+
+_HOMEPAGE = "https://nlp.biu.ac.il/~lazary/fh/"
+
+_LICENSE = "MIT"
+
+_URLs = {
+    "identification": {
+        "train": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/identification/processed/train.tsv",
+        "test": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/identification/processed/test.tsv",
+        "dev": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/identification/processed/dev.tsv",
+    },
+    "resolution": {
+        "train": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/resolution/processed/nfh_train.jsonl",
+        "test": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/resolution/processed/nfh_test.jsonl",
+        "dev": "https://raw.githubusercontent.com/yanaiela/num_fh/master/data/resolution/processed/nfh_dev.jsonl",
+    },
+}
+
+
+class NumericFusedHeads(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class NumericFusedHead(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `numeric_fused_head`